src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -162,14 +162,18 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
                     options["branded"] === true
                         ? undefined
                         : options["branded"],
-                generateTestProject:
-                    options["generateTestProject"] === true
+                "generate-sample-project":
```

Comment:
could you help to explain why the logic of generate-sample-project & generate-test-project are different?

Reply:
because default value of generate-sample-project is true, but generate-test-project is false.